final/conclusion.tex

```diff
@@ -1,6 +1,36 @@
 \chapter{Conclusion}
 \label{cha:conclusion}
 
+Over the course of the last quarter our project team has explored the Spoofax
+language workbench and created a read-eval-print-loop that operates with any
+language defined in the Spoofax workbench.
+
+During the research phase of the project it became apparent that the project
+team needed to gain quite extensive knowledge about both the conceptual ideas
+of language transformation and the Spoofax API implementing these concepts
```

Comment:
Yes, but nowhere in the report do we talk about it in that way, let alone introduce that concept. Nor have I seen this concept anywhere else in spoofax docs/papers.